.changeset/six-coats-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Added serve_directly option to assets
```

Comment:
The format is a type followed by a present tense description of what is changing from the point of view of someone reading the changelog.

```suggestion
feat: add `serve_directly` option to Workers with Assets

Users can now specify whether their assets are served directly against HTTP requests or whether these requests always go to the Worker, which can then respond with asset retrieved by its assets binding.
```

Something like that.